xml/System.Drawing.Imaging.Effects/TintEffect.xml

```diff
@@ -58,10 +64,15 @@
         <Parameter Name="amount" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="hue">To be added.</param>
+        <param name="hue">Integer in the range of 0 to 360 that specifies the hue to be strengthened or weakened.</param>
         <param name="amount">To be added.</param>
         <summary>To be added.</summary>
```

Comment:
@lonitra can you please provide descriptions for amount and the summary?

Reply:
Sure thing I can manually fix this. This is supposed to inherit docs from the other constructor for TintEffect which already has documentation on this in source. Curious if I did something wrong when porting then?

Reply:
> Curious if I did something wrong when porting then?

Not at all.

If it's supposed to have inheritdoc, then apologies, I didn't notice. There's nothing for you to do. Those 2 elements can remain undocumented if you're ok with inheriting the doc from that other API.